conf/layer.conf

```diff
@@ -9,5 +9,5 @@ BBFILE_COLLECTIONS += "pine64-layer"
 BBFILE_PATTERN_pine64-layer = "^${LAYERDIR}/"
 BBFILE_PRIORITY_pine64-layer = "5"
 
-LAYERSERIES_COMPAT_pine64-layer = "thud warrior"
+LAYERSERIES_COMPAT_pine64-layer = "thud"
```

Comment:
Why remove warrior?

Reply:
because this change is for the thud branch only